Update troubleshooting.md (#509)
pupil-labs/pupil-docs: src/invisible/troubleshooting.md

1. Are you using a **OnePlus 6** device?
   - Make sure the **Companion app is locked** to the Android app overview. If it is not locked, you should see a red lock symbol in the top-left of the companion app's home screen. You can find instructions on how to lock the app [here](/invisible/user-guide/invisible-companion-app/#enable-application-lock "Pupil Invisible Companion OnePlus App Lock").

#### 'Always open Invisible Companion when...' dialog keeps opening when I plug in the Glasses!

> This is an Android bug that requires a defaults reset of the Pupil Invisible Companion App:

Reviewer: ```suggestion
This is an Android OS bug. Fix it by resetting Pupil Invisible Companion App to default settings:
```

> 1. Long press on the Pupil Invsible Companion App in the home screen.

Reviewer: ```suggestion
1. Long press on the Pupil Invisible Companion App icon in the home screen.
```

1. Click `App Info` then `Advanced` followed by `Open by default`. 
1. Click `Clear defaults`. (No recordings will be lost.)

> 1. Open the App and follow the dialoges.

Reviewer: ```suggestion
1. Open the App & follow along with the prompts.
```

Reviewer: This part is a bit unclear. You mean accept permission prompts?

Reviewer 2: Yes, this would be about the individual USB device permissions that have previously been reset

```suggestion
1. Open the App and connect your Pupil Invisible device. Tick the "Always open Invisible Companion when..." checkboxes and accept the permission requests. Now they will be saved properly.
```

      device = Device(address=ip, port="8080")
      ```

>    - Alternatively, you can circumvent this by running a separate WiFi using the phone's hot spot functionality or an extra router.

Reviewer: ```suggestion
   - Alternatively, you can circumvent this by running a separate WiFi using the phone's hotspot functionality or a dedicated WiFi router.
```
